README.md

```diff
@@ -15,10 +15,10 @@ avoiding custom forks and binary solutions.
 This layer depends on:
 
 * URI: git://github.com/openembedded/openembedded-core
-  * branch: master
+  * branch: thud
   * revision: HEAD
 * URI: git://github.com/openembedded/bitbake
-  * branch: master
+  * branch: 1.42
```

Comment:
I don't test on non master branch so I don't think changing the README is a good idea

Reply:
when building on the thud branch pointing to master doesn't work. However using 1.42 does.